contracts/bancorx/BancorX.sol

```diff
@@ -34,395 +34,577 @@ contract BancorX is IBancorX, TokenHolder, ContractRegistryClient {
-    uint256 public prevLockBlockNumber; // the block number of the last lock transaction
-    uint256 public prevReleaseBlockNumber; // the block number of the last release transaction
-    uint8 public minRequiredReports; // minimum number of required reports to release tokens
+    uint256 private _maxLockLimit; // the maximum amount of tokens that can be locked in one transaction
+    uint256 private _maxReleaseLimit; // the maximum amount of tokens that can be released in one transaction
+    uint256 private _minLimit; // the minimum amount of tokens that can be transferred in one transaction
+    uint256 private _prevLockLimit; // the lock limit *after* the last transaction
+    uint256 private _prevReleaseLimit; // the release limit *after* the last transaction
+    uint256 private _limitIncPerBlock; // how much the limit increases per block
+    uint256 private _prevLockBlockNumber; // the block number of the last lock transaction
+    uint256 private _prevReleaseBlockNumber; // the block number of the last release transaction
+    uint8 private _minRequiredReports; // minimum number of required reports to release tokens
 
-    IERC20 public override token; // erc20 token
+    IERC20 private _token; // erc20 token
 
-    bool public xTransfersEnabled = true; // true if x transfers are enabled, false if not
-    bool public reportingEnabled = true; // true if reporting is enabled, false if not
+    bool private _xTransfersEnabled = true; // true if xTransfers are enabled, false if not
+    bool private _reportingEnabled = true; // true if reporting is enabled, false if not
 
     // txId -> Transaction
-    mapping(uint256 => Transaction) public transactions;
+    mapping(uint256 => Transaction) private _transactions;
 
     // xTransferId -> txId
-    mapping(uint256 => uint256) public transactionIds;
+    mapping(uint256 => uint256) private _transactionIds;
 
     // txId -> reporter -> true if reporter already reported txId
-    mapping(uint256 => mapping(address => bool)) public reportedTxs;
+    mapping(uint256 => mapping(address => bool)) private _reportedTxs;
 
     // address -> true if address is reporter
-    mapping(address => bool) public reporters;
+    mapping(address => bool) private _reporters;
 
     /**
      * @dev triggered when tokens are locked in smart contract
      *
-     * @param _from    wallet address that the tokens are locked from
-     * @param _amount  amount locked
+     * @param from wallet address that the tokens are locked from
+     * @param amount amount locked
      */
-    event TokensLock(address indexed _from, uint256 _amount);
+    event TokensLock(address indexed from, uint256 amount);
 
     /**
      * @dev triggered when tokens are released by the smart contract
      *
-     * @param _to      wallet address that the tokens are released to
-     * @param _amount  amount released
+     * @param to wallet address that the tokens are released to
```

Comment:
maybe 'target' instead of 'to'?

Reply:
Sure

Reply:
TBH, maybe not worth the effort, so consider it accordingly...